examples/ex_filtering/ex_filtering.pro

```diff
@@ -0,0 +1,111 @@
+#--------------------------------------------------------------------------------------------------------------
+#
+# @file     ex_filtering.pro
+# @author   Ruben Doerrfel <ruben.doerfel@tu-ilmenau.de>;
+#           Matti Hamalainen <msh@nmr.mgh.harvard.edu>
+# @version  1.0
+# @date     11, 2019
+#
+# @section  LICENSE
+#
+# Copyright (C) 2019, Ruben Doerrfel and Matti Hamalainen. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without modification, are permitted provided that
+# the following conditions are met:
+#     * Redistributions of source code must retain the above copyright notice, this list of conditions and the
+#       following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and
+#       the following disclaimer in the documentation and/or other materials provided with the distribution.
+#     * Neither the name of MNE-CPP authors nor the names of its contributors may be used
+#       to endorse or promote products derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED
+# WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+# PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+#
+# @brief    This project file generates the makefile to build the ex_filtering example.
+#
+#--------------------------------------------------------------------------------------------------------------
+
+include(../../mne-cpp.pri)
+
+TEMPLATE = app
+
+QT += widgets 3dextras
+
+CONFIG   += console
+CONFIG   -= app_bundle
+
+TARGET = ex_filtering
+
+CONFIG(debug, debug|release) {
+    TARGET = $$join(TARGET,,,d)
+}
+
```

Comment:
Add the following for static build support:
```
DESTDIR =  $${MNE_BINARY_DIR}

contains(MNECPP_CONFIG, static) {
    CONFIG += static
    DEFINES += STATICLIB
}
```
Check this [test_fiff_rwr.pro](https://github.com/mne-tools/mne-cpp/blob/master/examples/ex_read_raw/ex_read_raw.pro) file.